about.html

```diff
@@ -0,0 +1,128 @@
+            <li><a href="#">Mission</a></li>
+            <li><a href="#">Plan</a></li>
+            <li id="consul-lg"><a href="#">Consul</a></li>
+          </ul>
+        </nav>
+      </div>
+
+    </header>
+    <main>
+      <section class="about-section">
+        <div class="about-wrapper">
+        <div class="about-section-header">
+          <h2 class="about-greeting">"Hello! Sharing World"</h2>
+          <h1 class="about-info">Unlocking the potential of those who advance the world</h1>
+        </div>
+        <div class="about-text">
+          <p>
+            Get rid of the hassle of starting from scratch with business consulting website template, Consul. It’s a tool of many great advantages, starting with a professional design. You can create a powerful website by using Consul as-is, but that’s not necessary.
+          </p>
+        </div>
+
+        <div class="about-contact">
+          <p>Please contact us per Email for any further questions about Consul</p>
+          <a href="#" class="about-adress">
+            consul@gmail.com
+          </a>
+        </div>
+      </div>
+      </section>
+      <section class="about-details">
+        <div class="about-details-wrapper">
+          <h2 class="about-details-title">The gain insights, advice and tools to achieve your mission-critical priorities</h2>
+          
+
+          <p>The logo of Consul was decided through the logo competition from 8 june to 7 july</p>
+        </div>
+
+        <div class="about-details-img">
+          <img src="./img/170-1705916_hashicorp-consul.png" alt=" Global ">
+        </div>
+      </section>
+      
+      <div class="about-middle-page">
+        <div class="about-middle-page-wrapper">
+          <h2>See the past of Consul</h2>
+          <p>We help leading organizations succed with their most critical</p>
+        </div>
+      </div>
+
+      <div class="about-images">
+        <div class="about-images-img">
+          <div class="about-img-pic"><img src="./img/Angela-Shante_NoisyClassroom-300x212.jpg" alt=" Author"></div>
+          <div class="about-img-color"></div>
+          <div class="about-img-text">SMILE ALWAYS</div>
+        </div>
+        <div class="about-images-img">
+          <div class="about-img-pic"><img src="./img/c2.jpeg" alt=" Author"></div>
+          <div class="about-img-color"></div>
+          <div class="about-img-text">A BETTER BUSINESS PLAN</div>
+        </div>
+      </div>
+      <footer>
+        <div class="footer-header">
+          <h2>Partner</h2>
+        </div>
+        <ul class="footer-refs">
+          <li><a href="#"><img src="./img/logo1.png" alt=""></a></li>
+          <li><a href="#"><img src="./img/logo2.png" alt=""></a></li>
+          <li><a href="#"><img src="./img/logo5.png" alt=""></a></li>
+          <li><a href="#"><img src="./img/logo6.png" alt=""></a></li>
+          <li><a href="#"><img src="./img/logo9.png" alt=""></a></li>
+        </ul>
```

Comment:
- [ ] Looking at the original design, the `Partner` section should be completely removed from the `About` page.
- [ ] The footer section of the `About` page should have a dark background.